GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java

```diff
@@ -193,6 +194,19 @@ public GVRActivity getActivity() {
     }
 
     /**
+     * Get the {@link GVRImporter} to use for loading assets.
+     * 
+     * A {@code GVRImporter} loads models asynchronously from your applications's
+     * local storage or the network.
+     * 
+     * 
+     * @return The asset loader associated with this context.
+     */
+    public GVRImporter getAssetLoader() {
```

Comment:
I will rename it GVRAssetLoader then. I think that is a better name. It was a private class until now so nobody was using it externally.

From: rahul [mailto:notifications@github.com]
Sent: Thursday, June 2, 2016 5:42 PM
To: Samsung/GearVRf <GearVRf@noreply.github.com>
Cc: Nola Donato <nola.donato@samsung.com>; Author <author@noreply.github.com>
Subject: Re: [Samsung/GearVRf] Wait for textures to be loaded before adding model to the scene (#597)


In GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java<https://github.com/Samsung/GearVRf/pull/597#discussion_r65642518>:

> @@ -193,6 +194,19 @@ public GVRActivity getActivity() {

>      }

>

>      /**

> +     * Get the {@link GVRImporter} to use for loading assets.

> +     *

> +     * A {@code GVRImporter} loads models asynchronously from your applications's

> +     * local storage or the network.

> +     *

> +     *

> +     * @return The asset loader associated with this context.

> +     */

> +    public GVRImporter getAssetLoader() {

The function description says that the call would return an asset loader but it returns an importer. We would decide on a name either rename the call to getImporter or rename GVRImporter to AssetLoader.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Samsung/GearVRf/pull/597/files/fb58b0721681e130ce48eec2b79e1fc44b7ecf78#r65642518>, or mute the thread<https://github.com/notifications/unsubscribe/AQw1S8-MZszN4PJdSdbOzfcpbZJrQigpks5qH3hjgaJpZM4ItA3y>.